Shareit v4.0

DpaNniK opened the pull request.

DpaNniK committed:
приложение разделено на 2 микросервиса

DpaNniK committed:
приложение разделено на 2 микросервиса

Alexander-Bolotov commented on server/src/main/java/ru/practicum/shareit/booking/BookingController.java:
```diff
@@ -17,15 +17,20 @@ public class BookingController {
+        booking.setBookerId(Math.toIntExact(bookerId));
+        booking.setStart(bookingDto.getStart());
+        booking.setEnd(bookingDto.getEnd());
+        booking.setItemId((int) bookingDto.getItemId());
```
конвертация должна происходить в маппере

DpaNniK replied:
done

DpaNniK committed:
Shareit V4.2
Из pom удалена зависимость с валидацией
В сервисе убраны проверки для границ пагинации
Исключения уточнены значениями, которые их вызывают

DpaNniK committed:
Shareit V4.2
Из pom удалена зависимость с валидацией
В сервисе убраны проверки для границ пагинации
Исключения уточнены значениями, которые их вызывают
